### PR TITLE
L1d3p1a2

### PR DESCRIPTION
**Pull request by RobbieBuhite**

### CODE REVIEW

File: kzz7yh-a08909/cod-ve07v1_kzz7yh-a08909.xml [hunk 1]
```diff
@@ -112,8 +113,8 @@
       <lb ed="#V" n="8"/>gratia tantum sumus cognitionis tue lumine consecuti. 
    </p>
                <p xml:id="kzz7yh-a08909-d1e206">
-                  <lb ed="#V" n="9"/>Respondeo a em per natura diuitreatus 
-      <lb ed="#V" n="10"/>ra exiam per natura. Dicit enim. bro 
+                  <lb ed="#V" n="9"/>Respondeo quod deus est cognoscibilis a  
+      creatu<lb ed="#V" n="10" break="no"/>ra etiam per naturam. Dicit enim.b.
```
Comment (jeffreycwitt) [inline kzz7yh-a08909/cod-ve07v1_kzz7yh-a08909.xml:117]: the b here is for "beatus dionysius" 

```suggestion
      creatu<lb ed="#V" n="10" break="no"/>ra etiam per naturam. Dicit enim beatus 
```

File: kzz7yh-a08909/cod-ve07v1_kzz7yh-a08909.xml [hunk 2]
```diff
@@ -230,8 +232,8 @@
       pu<lb ed="#V" n="96" break="no"/>rum esse: ergo deum esse est nobis principium per se notum. 
     </p>
                <p xml:id="kzz7yh-a08909-d1e431">
-                  <lb ed="#V" n="97"/>Contra Datuilibi brie ca Ientantum inuanuit 
-      prni<lb ed="#V" n="98" break="no"/>tst et aatcia: vcant deum 
+                  <lb ed="#V" n="97"/>Contra Damascenum libro pro.c.3. Intactum inuanuit
```
Comment (jeffreycwitt) [inline kzz7yh-a08909/cod-ve07v1_kzz7yh-a08909.xml:235]:
```suggestion
                  <lb ed="#V" n="97"/>Contra Damascenum libro primo capitulo 3. Intantum inuanuit 
```

File: kzz7yh-a09618/cod-ve07v1_kzz7yh-a09618.xml [hunk 3]
```diff
@@ -57,19 +57,19 @@
...
-      <lb ed="#V" n="105"/>tur tria. Primo vtrum per creaturas 
+      <lb ed="#V" n="103"/>de modo  
+      cogno<lb ed="#V" n="104" break="no"/>scendi deum per creaturam. Circa quod 
+      querun<lb ed="#V" n="105" break="no"/>tur tria. Primo vtrum per creaturas 
       possi<lb ed="#V" n="106" break="no"/>mus aliquo modo conferre de deo quid est. 
       Se<lb ed="#V" n="107" break="no"/>cundo vtrum per creaturas possimus cognoscere quod deus trinus 
       <lb ed="#V" n="108"/>et vnus est. 3osi intellectus beatus deum per creaturam cognoscem potest. 
      </p>
             <div xml:id="kzz7yh-a09618-Dd1e121">
                <head xml:id="kzz7yh-a09618-Hd1e123">Quaestio 1</head>
```
Comment (jeffreycwitt) [inline kzz7yh-a09618/cod-ve07v1_kzz7yh-a09618.xml:68]: Are we mssing a question title here?

File: kzz7yh-a09618/cod-ve07v1_kzz7yh-a09618.xml [hunk 4]
```diff
@@ -298,8 +298,8 @@
                   <cb ed="#P" n="a"/>
                   <lb ed="#V" n="1"/>des est argumentum non apparentium. Sed deum esse trinum et 
       <lb ed="#V" n="2"/>vnum est de articlis fidei: ergo per creaturas non apporet nobis. 
-      <lb ed="#V" n="3"/>Rndeo conssumus inuestigando 
-      cognoscer<lb ed="#V" n="4" break="no"/>po sumus ulucsigsuu cognocere deum 
+      <lb ed="#V" n="3"/>Respondeo quod per creaturas ductu rationis nationalis non
```
Comment (jeffreycwitt) [inline kzz7yh-a09618/cod-ve07v1_kzz7yh-a09618.xml:301]:
```suggestion
      <lb ed="#V" n="3"/>Respondeo quod per creaturas ductu rationis naturalis non 
```

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]
Pretty good.

Note that there are two files changed in this pull request article 1 and article, but I think this branch was intended just to be for article 2. 

Ideally, I only one file to be changed per branch/pull request. 

But this will be ok for this PR.

File: kzz7yh-a09618/cod-ve07v1_kzz7yh-a09618.xml (hunk 3 above)
Comment (RobbieBuhite) [reply kzz7yh-a09618/cod-ve07v1_kzz7yh-a09618.xml:68]: I don't see a place in the text that specifically says "quaestio x..." but the wording is similar to a traditional question opening.

**Pull request merged**